vue-components/stories/Popover.stories.ts

```diff
@@ -48,26 +58,27 @@ basic.argTypes = {
 export function all(): Component {
 	return {
 		components: { Popover, Icon, Button },
+		data() {
+			return {
+				positions: [ 'top', 'right', 'bottom', 'left',
+			'top-right', 'top-left', 'right-bottom', 'right-top',
+			'bottom-right', 'bottom-left', 'left-top', 'left-bottom'
+		],
+			};
+		},
 		template: `
 					<div>
-					<div style="margin-left: 40vw; margin-top: 30vh;">
-						<Popover :is-shown="true" :react-to-hover="false">
+					<div style="margin-left: 40vw; margin-top: 30vh;" v-for="value in positions">
+						<Popover :is-shown="true" :react-to-hover="false" :position="value">
 							<template v-slot:target>
 								<Button :iconOnly="true" aria-label="show hint">
 									<Icon type="info-outlined" size="medium" style="vertical-align: top;"/>
 								</Button>
 							</template>
 							<template v-slot:default>
 								<div>
-									Foo bar
+								current position <b><em>{{ value }}</em></b>. You can modify it in the "Controls" section.
```

Comment:
It probably shouldn't claim twelve times that I can modify the position in the controls, when that is actually false. :)

Reply:
ha, correct :)